### PR TITLE
Changing the paginated API default size and tests

### LINKED ISSUE #68: Make the paginated API parameters consistent across APIs
### Issue Description

Currently the paginated api parameters are different across the APIs making them consistent.

### Expected Behavior

The parameter names should be consistent across APIs

### Current Behavior

Post this ticket it will be same across all the APIs

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [ ] Medium
 - [x] Low

### Additional Information

[Include any additional context, logs, or details that might be helpful in addressing the issue.]

### Checklist

- [ ] I have read and followed the project's code of conduct.
- [ ] I have searched for similar issues before creating this one.
- [ ] I have provided all the necessary information to understand and reproduce the issue.
- [ ] I am willing to contribute to the resolution of this issue.

---

Thank you for contributing to our project! We appreciate your help in improving it.

### PR DESCRIPTION
**Pull request by vikhyat187**
Date: 19th Dec

Developer Name: Vikhyat

----

## Issue Ticket Number

-   Add the relevant issue ticket number here (e.g., Closes #68).

## Description

Making the paginated api params consistent across APIs, example the page, size variable used in diff apis have diff names, make them consistent.

**Documentation Updated?**

-   [ ] Yes
-   [x] No

_If your feature adds a new API, then documentation related to the feature like API-Contracts, and Data-models  must be updated._

**Breaking Changes**

-   [ ] Yes
-   [x] No

_If your feature introduces breaking changes or if something is missing, please mention the related issue tickets._

**Development Tested?**

-   [x] Yes
-   [ ] No

_Confirm whether the changes have been tested locally during development._

**Tested in Staging?**

-   [ ] Yes
-   [x] No

_Indicate whether the changes have been tested in the staging environment._

**Under Feature Flag**

-   [ ] Yes
-   [x] No

_Specify if the changes are currently under a feature flag._

**Database Changes**

-   [ ] Yes
-   [x] No

_Indicate whether the changes include modifications to the database._

## Test Coverage

Have updated the tests as well for the changes.

### CODE REVIEW

**Review by Ajeyakrishna-k — APPROVED** [review]

File: skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillsController.java [hunk 1]
```diff
@@ -39,9 +40,9 @@ public ResponseEntity<?> createSkill(@RequestBody(required = true) @Valid SkillD
 
     @GetMapping("/")
     public Page<SkillDTO> getAllSkills(
-            @RequestParam(value = "page", defaultValue = "0") int page,
-            @RequestParam(value = "size", defaultValue = "100") int size) {
-        Pageable pageable = PageRequest.of(page, size);
+            @RequestParam(value = "offset", defaultValue = "0", required = false) @Min(0) int offset,
```
Comment (satyam73) [inline skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillsController.java:43]: what's the point of @Min(0) if required is false

Comment (vikhyat187) [reply skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillsController.java:43]: Its for the validation that you cannot have < 0 for the input.
Required says its mandatory or optional

Comment (satyam73) [reply skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillsController.java:43]: got it

**Review by satyam73 — APPROVED** [review]
LGTM

**Pull request merged**